test/sharing/gist_loader_test.dart

```diff
@@ -344,13 +344,14 @@ class MyStatelessWidget extends StatelessWidget {
 ''';
 
 final processedSample = '''
-import 'package:flutter_web/material.dart';
-import 'package:flutter_web_ui/ui.dart' as ui;
+// Flutter code sample for
```

Comment:
typo?

Reply:
It's actually a bug in the snippet generator, the output of which was used for this test. Greg's fixing it in this PR: https://github.com/flutter/flutter/pull/44787